grafana-plugin/src/pages/integration/Integration.helper.ts

```diff
@@ -68,8 +68,7 @@ export const IntegrationHelper = {
     const hasTelegram =
       store.hasFeature(AppFeature.Telegram) && store.telegramChannelStore.currentTeamToTelegramChannel?.length > 0;
     const isMSTeamsInstalled = Boolean(store.msteamsChannelStore.currentTeamToMSTeamsChannel?.length > 0);
-    const connectedChannels = store.mattermostChannelStore.getSearchResult();
-    const isMattermostInstalled = store.hasFeature(AppFeature.Mattermost) && Boolean(connectedChannels && connectedChannels.length)
+    const isMattermostInstalled = Boolean(store.mattermostChannelStore.currentTeamToMattermostChannel?.length > 0);
```

Comment:
Can be just `store.mattermostChannelStore.currentTeamToMattermostChannel?.length > 0` (without `Boolean` part) and the result will be always the same